File: example/javalib/testing/6-test-grouping/foo/test/src/foo/WorldTests.java
package foo;

import static org.junit.Assert.assertTrue;

import org.junit.Test;

public class WorldTests {
  @Test
  public void world() throws Exception {
    System.out.println("Testing World");
    String result = new Foo().hello();
    assertTrue(result.endsWith("World"));
    Thread.sleep(1000);
    System.out.println("Testing World Completed");
  }
}

File: example/kotlinlib/testing/5-test-grouping/foo/test/src/foo/WorldTests.kt
package foo

import io.kotest.core.spec.style.FunSpec
import io.kotest.matchers.string.shouldEndWith

class WorldTests :
    FunSpec({
        test("world") {
            println("Testing World")
            val result = Foo().hello()
            result shouldEndWith "World"
            java.lang.Thread.sleep(1000)
            println("Testing World Completed")
        }
    })
